src/services/errors/interfaces/base-error.interface.ts

```diff
@@ -0,0 +1,5 @@
+export interface BaseError {
```

Comment:
nitpick: I'd probably pick a more concrete name for this one as `Base` seems to imply that all the other errors that we have derive from this one.

Since this seems to come from the Http side and it's only used as the payload for `HttpException` I'd either rename it to something like `HttpExceptionPayload` or maybe just a simple `Record` (and we remove this class).

What do you think?

Reply:
I was thinking in having a common error class for all errors in the CGW, and extend it for specific needs. But you're right, maybe we can start by having this payload just for HttpException and see if we can find an standard for all the error in the future (and inherit from it).

Reply:
`HttpExceptionPayload` seems a good name to me 👍🏻